node/node.go

```diff
@@ -356,42 +357,102 @@ func (node *Node) Start() error {
 	if len(allTopics) == 0 {
 		return errors.New("have no topics to listen to")
 	}
-	weighted := make([]*semaphore.Weighted, len(allTopics))
-	const maxMessageHandlers = 200
+
+	const (
+		maxMessageHandlers = 200
+		threshold          = maxMessageHandlers / 2
```

Comment:
threshold can be higher? like 1/4? Also the throttle delay can be shorter than 100 ms.

During consensus in mainnet, hundreds of consensus msg will be sent around the nodes. It's easily got to throttle when the leader need to receive hundreds of msgs. If delay for 100ms, The node may need to wait for 10s+ to process all of them. Maybe change to 10ms delay?

Reply:
Totally - these just my intuition estimates , hence need rapid iteration today to find the right parameters

Reply:
Also, can we give consensus msg a higher priority? Even if we throttle, if we are clogged with random p2p msgs and the consensus msgs can't be processed. We are screwed too. And I think that's what happened in the morning attack (no node is down or crash, just the consensus is not moving forward)

Reply:
that's the next step if this doesn't work but then I need to write more pervasive logic & changes, can do that in another PR if needed

Reply:
need to figure out how long does it take to process one message, so to help determine the slowdown.

Reply:
Not every long , around 500 milliseconds if not less